examples/ads.amp.html

```diff
@@ -906,6 +907,18 @@ <h2 id="yieldmo">Yieldmo</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="webediads">Webediads (private ad network)</h2>
```

Comment:
Sorry, but the test was only available with a french IP. You need to used a VPN to see the ad. Webediads is a private ad network, available only for our own company in France, but, for now, I've disabled the GeoIp targetting for you (the ad is now available everywhere). 

Also, if we see too many request from this page, we will need to remove quickly this example ad. 

I understand your point, you need to have a full working example, but for us it's not good because anybody can use this code and display our ad, and if we have too many request, our servers could be overloaded (ok... maybe it's a dramatical scenario 💣 ).